versions.props

```diff
@@ -15,15 +15,15 @@ com.squareup.okhttp3:* = 3.11.0
 com.squareup.retrofit2:* = 2.4.0
 io.dropwizard.metrics:metrics-core = 3.2.5
 io.dropwizard:dropwizard-* = 1.3.1
-io.reactivex.rxjava2:rxjava = 2.0.4
+io.reactivex.rxjava2:rxjava = 2.2.3
```

Comment:
@robert3005 how did this merge without requiring any lock file updates? Does this actually mean that all of these versions were unused? (I think rxjava was actually deleted a long time ago)

Reply:
verifydependencylocksrecurrent is broken right now - investigating a fix